moonbeam-types-bundle/README.md

```diff
@@ -3,20 +3,74 @@
 Exports npm package `moonbeam-types-bundle`, formated as per polkadot-js specification to use
 with the app or the api.
 
-## Exports
+## ⚠️Warning: Types deprecation⚠️
 
+Following runtime upgrade 900 (include substrate v0.9.11), types are now retrieved from the node, in
+a **camelCase** format.
+
+A **new version** has been released `moonbeam-types-bundle@2.0.0`.  
+
+The default export `typesBundle` has **been removed** in order to avoid confusion.  
+
+**2 new typesBundles** are available:
+
+* `import { typesBundlePre900 } from "moonbeam-types-bundle"` to use the new naming convention (Which will be the **default on PolkadotJS SDK**)
+* `import { typesBundleDeprecated } from "moonbeam-types-bundle"` to keep using old naming convention (This will break at runtime 1000)
```

Comment:
to keep using old naming convention **that isn't camel cased** (This will break at runtime 1000)